config/development.sphinx.conf

```diff
@@ -0,0 +1,152 @@
+
```

Comment:
/.bundle
.idea
config/database.yml
config/secrets.yml
public/uploads/
dump.rdb
config/*.sphinx.conf
db/sphinx
# Ignore all logfiles and tempfiles.
/log/*
!/log/.keep
/tmp